### DISCUSSION TITLE
Version controlling in map editors?

### DISCUSSION THREAD

**Discussion by astro-angelfish** [ppy/osu#29642] [Ideas]
Like `git` in code/text document versioning, I think it is also possible on mapping. I think it would make use on map comparison and modding. We could just make another 'branch' and try out another pattern. And could be merged by the creator with approval.

The modded parts could have similar concept to `patch` files and could be cool if they can be visualized in editors. Ranked maps also can no longer have merging actions like an archived github project.

**Comment by peppy** [comment]
It's an eventual goal
